### DISCUSSION TITLE
Update repository URL after GitLab -> GitHub migration

### DISCUSSION THREAD

**Discussion by nicolasskilleos** [codemagic-ci-cd/codemagic-docs#1463] [Q&A] [answered]
Hi, as it was already asked, we also recently moved our app from GitLab to GitHub.

We would like to keep the current config and environment variables and we tried to update the repository URL in our app but the corresponding button is disabled.
As you suggested, we created a new app with the right GitHub URL in order to help you to make the update on the old app.

Here are our apps :

Old one (need repository URL update) : https://codemagic.io/app/61f7f38ddd214890aac72ce2

New one (temp app, will be deleted after URL update) : https://codemagic.io/app/63f342e7783b2fd9186d7882

Thanks

**Comment by icarusdust** [comment, marked as answer]
Hi @nicolasskilleos, migrated your repo successfully. Please check and let me know in case of any questions.

**Reply by nicolasskilleos** [reply]
Thank you very much @icarusdust !
